plugins/decl_hdf5/tests/CMakeLists.txt

```diff
@@ -122,6 +122,17 @@ if("${BUILD_HDF5_PARALLEL}")
 	set_property(TEST decl_hdf5_mpi_07_C PROPERTY PROCESSORS 4)
 endif()
 
+# case datasets is defined as a regex.
+# The data is writing in //. This data is defined with a dataset_selection and a dataset that depends on a regex
+if("${BUILD_HDF5_PARALLEL}")
+	add_executable(decl_hdf5_mpi_08_C decl_hdf5_mpi_test_08.c)
+	target_link_libraries(decl_hdf5_mpi_08_C PDI::PDI_C MPI::MPI_C)
+	add_test(NAME decl_hdf5_mpi_08_C COMMAND "${RUNTEST_DIR}" "${MPIEXEC}" "${MPIEXEC_NUMPROC_FLAG}" 4 ${MPIEXEC_PREFLAGS} "$<TARGET_FILE:decl_hdf5_mpi_08_C>" ${MPIEXEC_POSTFLAGS})
+	set_property(TEST decl_hdf5_mpi_08_C PROPERTY TIMEOUT 15)
+	set_property(TEST decl_hdf5_mpi_08_C PROPERTY PROCESSORS 4)
+endif()
+
+
```

Comment:
do you think we could also have a much simpler test for regex, without parallel HDF5?